ci/test_narwhals.sh

```diff
@@ -23,10 +23,15 @@ rapids-pip-retry install -U -e . pytest-env "hypothesis>=6.131.7"
 rapids-logger "Check narwhals versions"
 python -c "import narwhals; print(narwhals.show_versions())"
 
+TESTS_TO_ALWAYS_SKIP_CUDF="test_date_lit[cudf]"
```

Comment:
Curious why we need to skip this test now

Reply:
I have disabled `pa.date32` support for time-being. This test hits that code-flow.

Reply:
Done